Extensions/find_inactives.js

```diff
@@ -68,8 +75,10 @@ XKit.extensions.find_inactives = new Object({
 	},
 
 	get_count: function() {
-		var count_text = $("#tabs").html().match(/Following.*Tumblr/gm)[0];
-		count_text = count_text.replace(/[^0-9\.]/g, '');
+		var count_text = $("#tabs").html();
+		count_text = count_text.replace(/[^0-9]/g, '');
```

Comment:
would prefer to do peopleCount = parseInt(count_text.replace(etc)) so that the operations are selfdocumenting